### DISCUSSION TITLE
The meaning of ikelifetime on moon and sun?

### DISCUSSION THREAD

**Discussion by haraldhh** [strongswan/strongswan#1356] [General]
I have a site-to-site VPN setup with IPv6 tunneled in IPv4. "Moon" is at my home and "Sun" is at the office and is the default route. The clients are configued as right=%any on "Sun" so Sun is unable to start the connection. 

Since the IKE negotiation can't be initiated from "Sun", Isn't it a good idea to have ikelifetime=60m on Sun and say 40m on Moon. This way Moon is the one keeping the connection alive. Or is "Sun" able to renegotiate IKE while the tunnel is up and 60m is fine on both?

**Comment by tobiasbrunner** [comment]
You have to differentiate between rekeying (happens inline on the existing IKE_SA, only for IKEv2) and reauthenticaiton (creates a new IKE_SA from scratch), see [here](https://docs.strongswan.org/docs/5.9/config/rekeying.html) for details.
